pytoda/datasets/protein_sequence_dataset.py

```diff
@@ -43,11 +90,9 @@ def __init__(
             filepaths (FileList): paths to .smi, .csv/.fasta/.fasta.gz file
                 with the sequences.
             filetype (str): From {.smi, .csv, .fasta, .fasta.gz}.
-            protein_language (ProteinLanguage): a protein language or child
-                object. Defaults to None.
-                NOTE: ProteinFeatureLanguage objects cannot be created auto-
```

Comment:
If this NOTE is still true, we should keep it.

Reply:
I do not know about the truth of this NOTE

Reply:
Well, I think it's true, since in the case that no proteinlanguage is given, we just have a call `self.protein_language = ProteinLanguage(...`

I think this is fine logic-wise (I wouldn't want to add any complexity there). But we should probably just keep the note to make users aware.